pkg/oc/cli/admin/diagnostics/cluster.go

```diff
@@ -13,6 +13,7 @@ import (
 	rbacclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/rbac/internalversion"
 
 	appsclient "github.com/openshift/client-go/apps/clientset/versioned"
+	securityv1client "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
```

Comment:
Was this needed?

Reply:
Yes, diagnostics depends on the `ReconcileSCC` options struct: https://github.com/openshift/origin/blob/master/pkg/oc/cli/admin/diagnostics/diagnostics/cluster/scc.go#L52